osbs/utils.py

```diff
@@ -216,7 +217,7 @@ def subprocess_check_output_with_retry(args, max_retries=0, backoff_factor=0):
 
 @contextlib.contextmanager
 def checkout_git_repo(git_uri, git_ref, git_branch=None, max_retries=GIT_MAX_RETRIES,
-                      backoff_factor=GIT_BACKOFF_FACTOR):
+                      backoff_factor=GIT_BACKOFF_FACTOR, depth=None):
     """
     Context manager to check out a git repository, yielding a string with the path
     for the cloned repository.
```

Comment:
It would be nice to add an entry for depth in the docstring

Reply:
Fixed.